index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav-bar">
+      <span class="name">Miguel</span>
+      <span class="menu-icon"><img src="images/menu.svg" alt="menu icon"></span>
+    </nav>
+  </header>
+  <section class="container">
+    <h2 class="greetings"> <span>Hey There. I’m Miguel</span></h2>
+    <p class="my-job-title"> <span>I’am a Software Developer</span></p>
+    <p class="my-job-description"><span>I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</span></p>
+    <div class="social-icon-links">
+      <a href="#"><img src="images/github.svg" alt="github icon"></a>
+      <a href="#"><img src="images/linkedin.svg" alt="linkedin icon"></a>
+      <a href="#"><img src="images/angellist.svg" alt="angellist icon"></a>
+      <a href="#"><img src="images/twitter.svg" alt="twitter icon"></a>
+      <a href="#"><img src="images/medium.svg" alt="medium icon"></a>
+    </div>
```

Comment:
- [ ] Kindly use a `<ul>` tag to group the list of icons, and `<li>` per icon. That is semantically correct. The use of these tags and the `<a>` tag for the icons will let you connect your social networks, using href.